src/advanced_dialog.cpp

```diff
@@ -592,6 +592,21 @@ double AdvancedDialog::DetermineGuideSpeed()
     }
     return sidRate;
 }
+
+// Floating point equality comparisons can be a problem due to rounding and trivial inequalities.
+// PercentChange can be used to see if a change is worth reacting to
+double AdvancedDialog::PercentChange(double oldVal, double newVal)
+{
+    double chg;
+    if (oldVal > 0)
+    {
+        chg = fabs(1.0 - newVal / oldVal);
+    }
+    else
+        chg = fabs(newVal);

```

Comment:
hmm, the negative case (oldVal < 0) seems odd -- probably won't happen in this pr but could be an unexpected result if somebody else wants to use this method. how about something like this:
```
    if (fabs(oldVal) < 0.0001)
        return 100. * newVal; // not meaningful, but avoids divide by zero
    return 100. * fabs(1. - newVal / oldVal);
```

Reply:
Good idea.  This was originally a private static function in camera.cpp but when I promoted it to advanced_dialog, I should have paid more attention to the general use cases.